internal/command/completion.go

```diff
@@ -0,0 +1,120 @@
+package command
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+
+	"github.com/simplesurance/baur/v3/internal/log"
+	"github.com/simplesurance/baur/v3/pkg/baur"
+)
+
+func mockGitCommitID() (string, error) {
```

Comment:
I don´t know about the naming here. What about `noGitCommit`, or something like that?

Reply:
yes, I also don't like the current naming much, `noGitCommit` is imho confusing